bulk_grades/api.py

```diff
@@ -345,6 +345,11 @@ def validate_row(self, row):
         if row['course_id'] != self.course_id:
             raise ValidationError(_('Wrong course id {} != {}').format(row['course_id'], self.course_id))
 
+    def status(self):
```

Comment:
It is not possible because of the parent class `CSVProcessor` from `super-csv` does not send back the line numbers. The defaults were `self.error_messages.keys()` and discard to all of the line number of the errors.

Reply:
Sure, but you can construct the messages in the view class without this change, by referencing `self.processor.error_messages` , right?

Reply:
You're are right. Just changed the code.